### PR TITLE
French translation

### PR DESCRIPTION
**Pull request by EstebanJean-PRO**
Translated directly from the en_us file. Some may seem inacurrate, though, because I included french wordplays and a reference other than Don Robertson's novel (though mine is still a book).

### CODE REVIEW

File: src/main/resources/assets/some_assembly_required/lang/fr_fr.json [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  "item.some_assembly_required.spread": "Pâte à Tartiner",
+  "item.some_assembly_required.bread_slice": "Tranche de Pain",
+  "item.some_assembly_required.toasted_bread_slice": "Tranche de Pain Grillée",
+  "item.some_assembly_required.apple_slices": "Tranches de Pomme",
+  "item.some_assembly_required.golden_apple_slices": "Tranches de Pomme Dorée",
+  "item.some_assembly_required.enchanted_golden_apple_slices": "Tranches de Pomme Dorée Enchantée",
+  "item.some_assembly_required.chopped_carrot": "Haché de Carotte",
+  "item.some_assembly_required.chopped_golden_carrot": "Haché de Carotte Dorée",
+  "item.some_assembly_required.chopped_beetroot": "Haché de Betterave",
+  "item.some_assembly_required.tomato_slices": "Tranches de Tomate",
```
Comment (Geolykt) [inline src/main/resources/assets/some_assembly_required/lang/fr_fr.json:30]: I only have small knowledge of french. But is the usage of `de` really appropriate here?
As I understand it it marks possesion, which isn't truely the case here and googling "Haché de Carotte" yields "Carottes Hachées", which sounds more true to me.

Comment (EstebanJean-PRO) [reply src/main/resources/assets/some_assembly_required/lang/fr_fr.json:30]: You're right, it does sound truer language-wise, but while it's phonetically better, my approach allows quantification with MC's object stacking: "un Haché de Carottes", "deux HachéS de Carottes", etc... Literal translation would give "one Chop of Carrots", "two ChopS of Carrots", and so on.
Of course I see no objection on any changes, if others find your suggestion better

**Comment by ochotonida** [comment]
Looks good, thanks alot!
I'm gonna merge this now since I plan on updating soon. My French isn't quite good enough to provide some input on Geolykt's suggestion, but you can always create a new pull request if you change your mind.

For code changes I usually prefer a separate PR, but in this case they're pretty minor so it doesn't really matter that much.

**Comment by EstebanJean-PRO** [comment]
I may change little things here and there in that file, as well as provide more code optimization, and I believe that's the only things I can do to help this mod into 1.18

**Pull request merged**